org.mixedrealitytoolkit.uxcomponents/Button/Prefabs/Experimental/Action Button Checkbox (Experimental).prefab

```diff
@@ -1,268 +0,0 @@
-%YAML 1.1
```

Comment:
Do we want to bring Action Button Checkbox into the regular set of prefabs?

Reply:
I would like to add it.  I didn't add it because it was not part of the original set of Release prefabs, I created the experimental version as I found a similar prefab in one of the scenes.  I don't mind updating the PR to include the Action Button Checkbox but don't want to add it unless we all agree on adding it.  So, Would @keveleigh, @AMollis, @david-c-kline agree on adding the Experimental/Action Button Checkbox to the set of Release buttons? I vote aye, How do you vote?

Reply:
Yes I don't see any reason not to add it

Reply:
I've moved the Experimental Action Button Checkbox prefab to be part of the release buttons in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/712/commits/f6498d3829d80a60847bc1446d40a1c8941ec3ab , resolving this conversation.